src/SDKs/DevTestLabs/Management.DevTestLabs/Microsoft.Azure.Management.DevTestLabs.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>1.3.0</Version>
```

Comment:
@ryanvog either use version or versionPrefix for specifying your version.
Also this is a breaking change, so you will need major version bump.